python/cudf/cudf/core/indexed_frame.py

```diff
@@ -2610,6 +2610,10 @@ def _reindex(
             index = cudf.core.index.as_index(
                 index, name=getattr(index, "name", self._index.name)
             )
+            if not df._index.is_unique and len(index):
```

Comment:
Do we have a tests where `cudf.Series([0, 1], index=[1, 1]).reindex([])` works correctly?

Reply:
What is the right behaviour? pandas-2 `raises` in that case (and returns the series unmodified if we just call `reindex()`)

Reply:
Updated with the correct behavior 👍